### ISSUE TITLE
Can I achieve auto insertion of three `

### ISSUE THREAD

**Issue by andreyorst** [Raimondi/delimitMate#263]
Hello, your plugin is really great, but one thing is bothering me. Previously I used vim auto pairs plugin, which contain a very useful feature, but it a bit buggy in neovim, so I've switched to your plugin and now I miss that feature.

The point is when I edit markdown files, which I write a lot, I use code blocks:
![image](https://user-images.githubusercontent.com/19470159/31311033-863f75f6-abac-11e7-811a-263b6ea4a9ad.png)
which is basically is pair of ``` strings.

How it worked in auto pairs:
```
Input:  `|
Output: `|`
Input:  ``|`
Output: ``| (exit from `` pair)
Input:  ```|
Output: ```|```
```
How it works in your plugin (by default):
```
Input:  `|
Output: `|`
Input:  ``|`
Output: ``| (exit)
Input:  ```|
Output: ``|`
Input:  ```|`
Output: ```| (exit)
Input:   ````|
Output: ````| (huh?)
Input:  `````|
Output: ````|`
Input:  `````|`
Output: `````| (exit)
Input:  ``````|
Output: ``````| (?)
```
So basically instead of pressing <kbd>`</kbd> three times  i need to press it 8 times to get same thing, and then go back 3 times.
Can i have same behaviour with your plugin? I've checked helptags, but didn't find anything like that.

**Comment by Raimondi** [comment]
Have you tried `:help 'delimitMate_nesting_quotes'`?

**Comment by andreyorst** [comment]
Oh, thanks a lot, how did i missed that
